ruby/app.rb

```diff
@@ -7,17 +7,25 @@
 require 'json'
 require 'plaid'
 require 'sinatra'
+require 'pp'
```

Comment:
what is this?

Reply:
this had to do with pretty pretty print.  I don't think it is necessary anymore. I will delete.